### PR TITLE
go fuck(2)

### PR DESCRIPTION
**Pull request by margothequeen**
wanna fuck or hug

### CODE REVIEW

File: fuck.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func main() {
-    fmt.Println("let us fuck")
+    fmt.Println("i love 7586")
```
Comment (24eeeee) [inline fuck.go:9]: fuck

File: fuck.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func main() {
-    fmt.Println("let us fuck")
```
Comment (24eeeee) [inline fuck.go:9]: hug

**Review by 24eeeee — CHANGES_REQUESTED** [review]
aboba

File: fuck.go (hunk 1 above)
Comment (margothequeen) [reply fuck.go:9]: fuck

File: fuck.go (hunk 2 above)
Comment (margothequeen) [reply fuck.go:9]: hug

**Review by 24eeeee — APPROVED** [review]

**Pull request merged**